public/javascript/posts/BookAppointmentModal.js

```diff
@@ -149,7 +155,7 @@ angular.module('BookAppointmentApp')
       },
 
       paymentSelected: function(){
```

Comment:
whereever this function - `paymentSelected` is selected please use - `paymentSelected()` so as to differenciate the difference between the variable and function